tests/webapp/api/test_bugzilla.py

```diff
@@ -0,0 +1,52 @@
+import json
+
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from rest_framework.test import APIClient
+
+
+# import responses
+# import re
+# from django.conf import settings
+
+
+def test_create_bug(webapp, eleven_jobs_stored):
+    """
+    test creating a bug in bugzilla
+    """
+
+    # def request_callback(request):
+    #    headers = {}
+    #    return(200, headers, {"success":"123456"})
+    # url_re = re.compile(r'https://bugzilla-dev.allizom.org/.*')
+    # responses.add_callback(
+    #    responses.POST, url_re,
+    #    callback=request_callback, content_type="application/json",
+    # )
+    # responses.add(
+    #    responses.POST, url_re,
+    #    body='{"success":"123456"}', status=200, content_type="application/json",
+    # )

```

Comment:
Looks like this works:

    content = json.dumps({"success":"123456", "id": 323})
    responses.add(responses.POST,
                  'https://bugzilla-dev.allizom.org/rest/bug',
                  body=content, status=200,
                  match_querystring=False,
                  content_type='application/json')